src/simulator/perftest_report.py

```diff
@@ -90,9 +90,6 @@ def report(config: ReportConfig, df: pd.DataFrame):
 
 
 def lookup_periods(config):
-    if config.preserve_time:
```

Comment:
We always want to load the periods of the test because this is used other reporting functions.